style.css

```diff
@@ -0,0 +1,852 @@
+*,
+html {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  padding: 30px 10px;
+  align-items: center;
+}
+
+header a {
+  font-family: "Roboto Serif", serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 18px;
+  line-height: 24px;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+}
+
+.home {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 20px;
+  gap: 12px;
+  width: 100%;
+  height: auto;
+  background: #1c1a19;
+  background-image: url(images/Group.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.headline h1 {
+  font-family: "Crete Round", serif;
+  font-size: 56px;
+  font-weight: 400;
+  font-style: normal;
+  letter-spacing: 0.37px;
+  text-align: left;
+  color: #ff6b00;
+  margin-bottom: 0;
+}
+
+.headline h2 {
+  font-weight: 700;
+  font-size: 24px;
+  color: #fff5e1;
+  display: flex;
+  align-items: center;
+  flex-grow: 0;
+  font-family: "Roboto Serif", serif;
+  font-style: normal;
+}
+
+.headline p {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  font-size: 16px;
+  font-style: normal;
+  height: 123px;
+  color: #fff;
+  margin: 10px 0 0 0;
+}
+
+nav {
+  display: none;
+}
+
+.social-icons-des {
+  display: none;
+}
+
+.social-icons3 {
+  display: none;
+}
+
+.social-icons ul {
+  display: flex;
+  justify-content: space-between;
+  width: 200px;
+  height: 50px;
+  list-style: none;
+  align-items: center;
+}
+
+#shape1 {
+  position: absolute;
+  width: 20%;
+  top: 580px;
+  right: 0;
+}
+
```

Comment:
- [ ] As requested by the previous code reviewer, your project should be responsive on all screen sizes but currently, you have extra white spaces at the right at come screen sizes as you can see in the video below :point_down: Make sure you adjust the widths and align containers correctly to avoid this to make your project responsive :+1: :100: 
  
  [Screencast from 05-06-2023 03:21:08 PM.webm](https://user-images.githubusercontent.com/92303891/236633183-9b222383-7a4e-41df-8655-0e9694b8235e.webm)